### PR TITLE
Update to Xcode 11.3, Swift 5.0 (from 4.2)

### PR DESCRIPTION
**Pull request by muhasturk**
Update the Circle CI script and the common.xcconfig.

To fix https://github.com/mozilla-mobile/firefox-ios/issues/5915

### CODE REVIEW

File: megazords/ios/MozillaAppServices.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -48,6 +48,8 @@
 			compilerSpec = com.apple.compilers.proxy.script;
 			filePatterns = "*.proto";
 			fileType = pattern.proxy;
+			inputFiles = (
+			);
```
Comment (garvankeeley) [inline megazords/ios/MozillaAppServices.xcodeproj/project.pbxproj]: Can you remove all changes to this file? They aren't necessary.

File: megazords/ios/MozillaAppServices.xcodeproj/project.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings [hunk 2]
```diff
@@ -3,6 +3,8 @@
 <plist version="1.0">
 <dict>
 	<key>BuildSystemType</key>
-	<string>Original</string>
+	<string>Latest</string>
+	<key>PreviewsEnabled</key>
+	<false/>
```
Comment (garvankeeley) [inline megazords/ios/MozillaAppServices.xcodeproj/project.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings]: The change isn't needed.

File: megazords/ios/MozillaAppServices.xcodeproj/xcshareddata/xcschemes/MozillaAppServices.xcscheme [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Scheme
-   LastUpgradeVersion = "1010"
+   LastUpgradeVersion = "1130"
```
Comment (garvankeeley) [inline megazords/ios/MozillaAppServices.xcodeproj/xcshareddata/xcschemes/MozillaAppServices.xcscheme]: Changes to this file aren't necessary

File: xcconfig/common.xcconfig [hunk 4]
```diff
@@ -13,7 +13,7 @@ DYLIB_CURRENT_VERSION = 1
 DYLIB_INSTALL_NAME_BASE = @rpath
 LD_RUNPATH_SEARCH_PATHS = @executable_path/Frameworks @loader_path/Frameworks
 
-SWIFT_VERSION = 4.2
+SWIFT_VERSION = 5.0
```
Comment (garvankeeley) [inline xcconfig/common.xcconfig]: Nice, this is the correct place to make build config changes.

**Comment by garvankeeley** [comment]
Unnecessary make patches bigger and they have risk of confusing future contribs who think those changes are needed. Each patch forms a precendent for future patches, and contribs (particularly those with no familiarity with Xcode) can see easily what changes are needed to produce a given effect.

**Comment by garvankeeley** [comment]
@eoger this is ready to land

**Comment by eoger** [comment]
Landed in 612e56601eba43ddceb3ee3453ace2bc7e38fa72, thanks so much @muhasturk and @garvankeeley!

**Pull request closed without merging**